docs/_articles/en/user-guide/org-browser.md

```diff
@@ -17,6 +17,7 @@ The Org Browser displays the available metadata types and their corresponding co
 6. Refresh components for metadata type
 7. Retrieve source for metadata component
 8. Retrieve source for all components of metadata type
+9. Retrieve source for metadata component and open it on VSCode
```

Comment:
9. Retrieve source for metadata component and open it in VS Code.

Reply:
Done! 
I'm not using the final point to keep the coherence with the previous bullets.
@ekapner Let me know if it's fine

Reply:
Yes! @LisandroFernandezSF - no ending period needed. Whoops.